### ISSUE TITLE
Multiple Teams

### ISSUE THREAD

**Issue by loofou** [Vaelor/errbot-mattermost-backend#34]
Hello,

I can't seem to find a way to connect the bot to multiple teams. Is this even supported?
I had hoped the bot would recognize the teams it is already in, but apparently only the team in the errbot config gets recognized for the bot.

Thanks,
loofou

**Comment by Vaelor** [comment]
Hey! Multiple teams is something I had while initially developing the bot. I ran into a few problems during this time and had some concerns regarding privacy. Having the same bot in multiple Teams, would theoretically mean he has access to information of the other team, which could be something one does not want.
This, and the fact that in the api requests to mattermost you often have to specify a team I'd makes it quite difficult. Assume we have two teams, a bot and a auser, both are members kf both teams. You tell the bot over a direct message to join a room. This room exists on both teams, how would the bot know which of both rooms to join? IMHO this makes a lot of things very complicated, that's why I dismissed it very early in developing. And I haven't planned to support it to be honest. If there is a PR for it, as a feature to activate I would gladly accept it :-D
